examples/training/classification/basic_imagenet_training.json

```diff
@@ -0,0 +1,25 @@
+{
+    "script_authors": ["ianjjohnson"]
```

Comment:
I do like the idea of "authors" to attribute credit to the entire group that wrote it.